lib/nine_one_one.rb

```diff
@@ -2,47 +2,36 @@
 require_relative './nine_one_one/http'
 require_relative './nine_one_one/version'
 require_relative './nine_one_one/configuration'
+require_relative './nine_one_one/notifier'
 require_relative './nine_one_one/log_service'
 require_relative './nine_one_one/slack_service'
 require_relative './nine_one_one/pager_duty_service'
 
 module NineOneOne
-  def self.emergency_service
-    if config.send_pagers
-      PagerDutyService.new(config.pager_duty_integration_key)
-    else
-      LogService.new(config.logger)
-    end
-  end
-
-  def self.notification_service
-    if config.slack_enabled
-      SlackService.new(config.slack_webhook_url, username: config.slack_username,
-                                                 channel: config.slack_channel)
-    else
-      LogService.new(config.logger)
-    end
-  end
```

Comment:
This change is not backwards compatible. We are using those methods in Mjollnir:
https://github.com/u2i/mjollnir/blob/6b4cf1491863175980434f697a64bc6cee47b2ba/lib/mjollnir/configuration.rb#L75
https://github.com/u2i/mjollnir/blob/6b4cf1491863175980434f697a64bc6cee47b2ba/lib/mjollnir/incident_reporting.rb#L18

Personally, I like them because you can easily do a dependency injection using factory methods.

Reply:
Good catch, I'll bring them back